vendor.conf

```diff
@@ -10,59 +10,55 @@ github.com/containers/storage v1.8
 github.com/hashicorp/go-multierror v1.0.0
-github.com/hashicorp/go-cleanhttp v0.5.0
 github.com/imdario/mergo v0.3.6
 github.com/mattn/go-runewidth v0.0.4
 github.com/mattn/go-shellwords v1.0.3
 github.com/Microsoft/go-winio v0.4.11
 github.com/Microsoft/hcsshim v0.8.3
 github.com/mistifyio/go-zfs v2.1.1
 github.com/moby/moby f8806b18b4b92c5e1980f6e11c917fad201cd73c
-github.com/mtrmac/gpgme master
-github.com/Nvveen/Gotty master
+github.com/mtrmac/gpgme b2432428689ca58c2b8e8dea9449d3295cf96fc9
+# TODO: Gotty has not been updated since 2012. Can we find a replacement?
+github.com/Nvveen/Gotty cd527374f1e5bff4938207604a14f2e38a9cf512
 github.com/opencontainers/go-digest c9281466c8b2f606084ac71339773efd177436e7
 github.com/opencontainers/image-spec v1.0.0
-github.com/opencontainers/runc master
+github.com/opencontainers/runc v1.0.0-rc6
 github.com/opencontainers/runtime-spec v1.0.0
 github.com/opencontainers/runtime-tools v0.8.0
 github.com/opencontainers/selinux v1.1
 github.com/openshift/imagebuilder a4122153148e3b34161191f868565d8dffe65a69
 github.com/ostreedev/ostree-go 9ab99253d365aac3a330d1f7281cf29f3d22820b
-github.com/pborman/uuid master
-github.com/pkg/errors master
+github.com/pkg/errors v0.8.1
 github.com/pquerna/ffjson d49c2bc1aa135aad0c6f4fc2056623ec78f5d5ac
-github.com/seccomp/containers-golang master
-github.com/sirupsen/logrus master
-github.com/syndtr/gocapability master
-github.com/tchap/go-patricia master
+github.com/seccomp/libseccomp-golang v0.9.0
+github.com/seccomp/containers-golang v0.1
+github.com/sirupsen/logrus v1.0.0
+github.com/syndtr/gocapability d98352740cb2c55f81556b63d4a1ec64c5a319c2
+github.com/tchap/go-patricia v2.2.6
 github.com/ulikunitz/xz v0.5.5
 github.com/urfave/cli 934abfb2f102315b5794e15ebc7949e4ca253920
 github.com/vbatts/tar-split v0.10.2
-github.com/xeipuuv/gojsonpointer master
-github.com/xeipuuv/gojsonreference master
-github.com/xeipuuv/gojsonschema master
-golang.org/x/crypto master
-golang.org/x/net master
-golang.org/x/sync 42b317875d0fa942474b76e1b46a6060d720ae6e
-golang.org/x/sys master
-golang.org/x/text master
+github.com/xeipuuv/gojsonpointer 4e3ac2762d5f479393488629ee9370b50873b3a6
+github.com/xeipuuv/gojsonreference bd5ef7bd5415a7ac448318e64f11a24cd21e594b
+github.com/xeipuuv/gojsonschema v1.1.0
+golang.org/x/crypto ff983b9c42bc9fbf91556e191cc8efb585c16908 https://github.com/golang/crypto
+golang.org/x/net 45ffb0cd1ba084b73e26dee67e667e1be5acce83 https://github.com/golang/net
+golang.org/x/sync 37e7f081c4d4c64e13b10787722085407fe5d15f https://github.com/golang/sync
+golang.org/x/sys 7fbe1cd0fcc20051e1fcb87fbabec4a1bacaaeba https://github.com/golang/sys
+golang.org/x/text e6919f6577db79269a6443b9dc46d18f2238fb5d https://github.com/golang/text
 gopkg.in/cheggaaa/pb.v1 v1.0.27
 gopkg.in/yaml.v2 v2.2.2
-k8s.io/apimachinery master
-k8s.io/client-go master
-k8s.io/kubernetes master
+k8s.io/client-go kubernetes-1.10.13-beta.0 https://github.com/kubernetes/client-go
```

Comment:
does this cover apimachinery?

Reply:
Is apimachinery needed? vndr stated it's not and a quick grep in the code didn't reveal a dependency.